client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -139,18 +140,47 @@ class BusinessDetails extends Component {
 		return keys( pickBy( values ) ).filter( name => this.extensions.includes( name ) );
 	}
 
-	getNumberRangeString( min, max = false ) {
+	convertCurrency( value ) {
+		// These are rough exchange rates from USD.  Precision is not paramount.
+		// The keys here should match the keys in `getCurrencyData`.
+		const exchangeRates = {
+			US: 1,
+			EU: 0.9,
+			IN: 71.24,
+			GB: 0.76,
+			BR: 4.19,
+			VN: 23172.5,
+			ID: 14031.0,
+			BD: 84.87,
+			PK: 154.8,
+			RU: 63.74,
+			TR: 5.75,
+			MX: 19.37,
+			CA: 1.32,
+		};
+
+		const region = getCurrencyRegion( this.props.settings.woocommerce_default_country );
+		const exchangeRate = exchangeRates[ region ] || exchangeRates.US;
+
+		return Math.round( value * exchangeRate / 1000 ) * 1000;
+	}
+
+	numberFormat( value ) {
+		return formatValue( 'number', value );
+	}
+
+	getNumberRangeString( min, max = false, format = this.numberFormat ) {
```

Comment:
@joshuatf: I had to add back in the `format` parameter that was removed in https://github.com/woocommerce/woocommerce-admin/commit/1b2fb37ada966aea43593977d9eaa91082f06d48. The product question uses it to format regular numbers, and the revenue question uses it to format currency.

Reply:
That makes sense- thanks for the explanation 😄